finetune/model.py

```diff
@@ -123,7 +136,7 @@ def _text_to_ids(self, *Xs, max_length=None):
         tokens, mask = self._array_format(token_idxs)
         return tokens, mask
 
-    def _finetune(self, *Xs, Y, batch_size=BATCH_SIZE):
+    def _finetune(self, *Xs, Y, batch_size=BATCH_SIZE, val_frac=0.05, eval_interval=150):
```

Comment:
rename to validation_size and allow int / float (we get this for free thanks to `train_test_split` being used)